src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -24,6 +24,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
```

Comment:
where is `ArrayList` used in this file?

Reply:
Same here... if the import is not used please remove it, thanks!